src/utils.js

```diff
@@ -219,6 +219,7 @@ export const convertToSlipData = (source, intl, timeZone, locale, slipName = 'Pi
       'requester.firstName': requester.firstName,
       'requester.lastName': requester.lastName,
       'requester.middleName': requester.middleName,
+      'requester.preferredFirstName': requester.preferredFirstName ? requester.preferredFirstName : requester.firstName,
```

Comment:
Could you please investigate can we change to  `requester.preferredFirstName || requester.firstName,`?